include/mender-utils.h

```diff
@@ -74,6 +74,11 @@ extern "C" {
  */
 #define MENDER_ARG_UNUSED __attribute__((unused))
 
+/**
+ * @brief A utility macro to make marking weak functions less noisy/disruptive
+ */
+#define MENDER_WEAK_FUNC __attribute__((weak))
```

Comment:
For consistency with the above, I would rewrite as

```suggestion
#define MENDER_FUNC_WEAK __attribute__((weak))
```